app/src/main/java/com/example/playlistmaker/SearchActivity.kt

```diff
@@ -66,6 +78,11 @@ class SearchActivity : AppCompatActivity() {
     private lateinit var searchHistoryClearButton: Button
     private lateinit var searchHistoryRecyclerView: RecyclerView
 
+    private var isClickAllowed = true
+    private val searchRunnable = Runnable { search(searchString) }
+    private var mainThreadHandler: Handler? = null
```

Comment:
Можно не nullable сделать, сразу создать:
```suggestion
    private var mainThreadHandler: Handler = Handler(Looper.getMainLooper())
```

Reply:
Ага, я даже про это спросить хотел можно ли инициализировать ДО вызова onCreate, спасибо!